docs/docs/EntitiesSaveStateVector.md

```diff
@@ -13,10 +13,166 @@ ___
+
+___
+### GetInitSeed () {: aria-label='Functions' }
+#### int GetInitSeed ( ) {: .copyable aria-label='Functions' }
+
+___
+### GetPos () {: aria-label='Functions' }
+#### [Vector](Vector.md) GetPos ( ) {: .copyable aria-label='Functions' }
+
+___
+### GetSpawnerType () {: aria-label='Functions' }
+#### int GetSpawnerType ( ) {: .copyable aria-label='Functions' }
+
+___
+### GetSpawnerVariant () {: aria-label='Functions' }
+#### int GetSpawnerVariant ( ) {: .copyable aria-label='Functions' }
+
+___
+### GetSubType () {: aria-label='Functions' }
+#### int GetSubType ( ) {: .copyable aria-label='Functions' }
+
+___
+### GetType () {: aria-label='Functions' }
+#### int GetType ( ) {: .copyable aria-label='Functions' }
+
+___
+### GetU1 () {: aria-label='Functions' }
+#### int GetU1 ( ) {: .copyable aria-label='Functions' }
+
+___
+### GetVariant () {: aria-label='Functions' }
+#### int GetVariant ( ) {: .copyable aria-label='Functions' }
+
+___
+### SetB1 () {: aria-label='Functions' }
+#### void SetB1 ( boolean bool ) {: .copyable aria-label='Functions' }
+
+___
+### SetB2 () {: aria-label='Functions' }
+#### void SetB2 ( boolean bool ) {: .copyable aria-label='Functions' }
+
+___
+### SetF1 () {: aria-label='Functions' }
+#### void SetF1 ( float num ) {: .copyable aria-label='Functions' }
+
+___
+### SetF2 () {: aria-label='Functions' }
+#### void SetF2 ( float num ) {: .copyable aria-label='Functions' }
+
+___
+### SetI1 () {: aria-label='Functions' }
+#### void SetI1 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetI2 () {: aria-label='Functions' }
+#### void SetI2 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetI3 () {: aria-label='Functions' }
+#### void SetI3 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetI4 () {: aria-label='Functions' }
+#### void SetI4 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetI5 () {: aria-label='Functions' }
+#### void SetI5 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetI6 () {: aria-label='Functions' }
+#### void SetI6 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetI7 () {: aria-label='Functions' }
+#### void SetI7 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetI8 () {: aria-label='Functions' }
+#### void SetI8 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetPos () {: aria-label='Functions' }
+#### void SetPos ( [Vector](Vector.md) TargetPosition ) {: .copyable aria-label='Functions' }
+
+___
+### SetSubType () {: aria-label='Functions' }
+#### void SetSubType ( int subType ) {: .copyable aria-label='Functions' }
+
+___
+### SetType () {: aria-label='Functions' }
+#### void SetType ( int type ) {: .copyable aria-label='Functions' }
+
+___
+### SetU1 () {: aria-label='Functions' }
+#### void SetU1 ( int num ) {: .copyable aria-label='Functions' }
+
+___
+### SetVariant () {: aria-label='Functions' }
+#### void SetVariant ( int variant ) {: .copyable aria-label='Functions' }
```

Comment:
Theses are normally documented on EntitiesSaveState, and they seem to be all documented already